assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -354,6 +354,12 @@
 					},
 				} );
 			} else {
+				// Override the onDrop callback with a user-provided one if present.
```

Comment:
It should work with a hook, please consult with @iNewLegend

Reply:
We're not using any command here, where the hook should be placed?